javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrintVisitor.java

```diff
@@ -1245,6 +1245,10 @@ public void visit(final SwitchEntry n, final Void arg) {
                     printer.print(", ");
                 }
             }
+            // `case null, default -> ...` added in JEP 441
+            if (n.getLabels().isNonEmpty() && n.isDefault()) {
+                printer.print(", default");
```

Comment:
Are you sure that the "default" case must be preceded by a comma?

Reply:
Same comment as above (this is the deprecated printer, but I figured I'd update it here too).